packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -121,6 +121,14 @@ export const components = ({
             outlineStyle: "solid",
             zIndex: 1,
           },
+          svg: {
```

Comment:
matching the fontsize set for the MuiLink so the svg are of same size when used inside the Accordion